From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Carlos Garnacho <carlosg@gnome.org>
Date: Tue, 15 Feb 2022 15:47:19 +0100
Subject: [PATCH] build: Drop catchsegv Meson dependency

This is basically mostly desirable for CI. Since that still
runs under .gitlab-ci/run-test.sh which wraps execution on
catchsegv, we presumably still have the positive effects there,
or at least until we update CI to a world where catchsegv is
gone.

Closes: https://gitlab.gnome.org/GNOME/mutter/-/issues/2120
Upstream-Status: Pending
Reference: https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2294
Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index d3a6bf0..57208cc 100644
--- a/meson.build
+++ b/meson.build
@@ -311,7 +311,6 @@ if have_tests
     ],
     exe_wrapper: [
       default_test_wrappers,
-      find_program('catchsegv'),
       find_program('xvfb-run'), '-a', '-s', '+iglx -noreset',
     ],
     timeout_multiplier: 10,
